SB2018/001-student-robotics-replacement-competition.md

```diff
@@ -1,6 +1,7 @@
 ---
 to: All teams
 subject: Student Robotics Replacement Competition
+send: 2017-11-06
```

Comment:
`send` rather than `sent` implies that the process is to add the files to this repo and _then_ send them on the given date. Is that the case?

Reply:
That is the case, yes, however I think `sent` here is still the right language

Reply:
Or go semi-ambiguous with just `date`. Which the more I think of it is actually the nicest

Reply:
If we're agreed on having the date in the file name, I'd rather not also include it inline -- the duplication seems likely to cause more issues than it helps with.